src/poetry/installation/wheel_installer.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import annotations
+
+import os
+import platform
+import sys
+
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from installer.destinations import SchemeDictionaryDestination  # type: ignore[import]
+from installer.sources import WheelFile  # type: ignore[import]
+
+from poetry import __version__
```

Comment:
```suggestion
from poetry.__version__ import __version__
```

I just noticed, that the version in the INSTALLER file is not printed correctly. We should fix this during the next rebase.